packages/design-system-react-native/src/components/Icons/Icon/Icon.constants.ts

```diff
@@ -17,10 +17,13 @@ export const TWCLASSMAP_ICON_ICONCOLOR: Record<IconColor, string> = {
   [IconColor.PrimaryDefault]: 'text-primary-default',
   [IconColor.PrimaryInverse]: 'text-primary-inverse',
   [IconColor.ErrorDefault]: 'text-error-default',
+  [IconColor.ErrorDefaultPressed]: 'text-error-defaultPressed',
   [IconColor.ErrorInverse]: 'text-error-inverse',
   [IconColor.SuccessDefault]: 'text-success-default',
+  [IconColor.SuccessDefaultPressed]: 'text-success-defaultPressed',
   [IconColor.SuccessInverse]: 'text-success-inverse',
   [IconColor.WarningDefault]: 'text-warning-default',
+  [IconColor.WarningDefaultPressed]: 'text-warning-defaultPressed',
   [IconColor.WarningInverse]: 'text-warning-inverse',
```

Comment:
@georgewrmarshall you might need to add these colors to the react side as well

Reply:
Good question. Can these be overridden using the Tailwind classname `text-error-default-pressed` in `twrnc`? If so, I’m not sure we need to include all the colors here, as the `color` prop is meant to provide safe defaults. These pressed colors are likely to be used less often, and if needed, they can be overridden with Tailwind classnames.

Reply:
@georgewrmarshall for now I'll add it to be safe